deploy/balena-k3s/bastion/Dockerfile

```diff
@@ -63,4 +63,4 @@ RUN echo "source /app/edge-endpoint/deploy/balena-k3s/bastion/src/kube-bash.sh"
 RUN chmod +x ./edge-endpoint/deploy/bin/setup-ee.sh
 
 ENTRYPOINT []
-CMD ["/bin/sh", "-c", "./edge-endpoint/deploy/bin/setup-ee.sh && tail -f /dev/null"]
```

Comment:
We have some hacky code in there now that starts the containers but then just sleeps if `RUN_EDGE_ENDPOINT` isn't set.

What I'd really like to do is reconfigure all of our Balena stuff to do as much as possible in k3s and then make these decisions dynamically. This would also have the advantage of letting us use fewer fleets by having the same docker-compose setup across many different configurations.

Reply:
I'll remove the autostart for edge endpoint to keep the existing behavior for now